src/compiler/crystal/tools/init/template/example.cr.ecr

```diff
@@ -1,6 +1,6 @@
-require "./<%= config.name %>/*"
-
 # TODO: Write documentation for `<%= module_name %>`
 module <%= module_name %>
+  VERSION = {{ `shards version #{__DIR__}`.chomp.stringify }}
```

Comment:
You can't just assume that `shards` is available, let alone in PATH. This defeats the whole point of decentralization. Do you really think Crystal libraries should literally not work if `shards` is not in PATH?

Reply:
In case it wasn't clear, this literally puts the line `VERSION = {{ `shards version #{__DIR__}`.chomp.stringify }}` into all new projects. It's not a dependency of `crystal init`, it's a dependency of every library.

Reply:
I'm not particularly fond of hard dependency on `shards` either, here it is IMHO justified.
`crystal init` is perhaps the first real compiler command run by novice developers following a Crystal tutorial. It should have all batteries included and provide an easy way into the Crystal ecosystem which includes `shards`.
The init command is opinionated but dedicated to provide an easy tool for developers to make them happy. If that doesn't fit with specific needs, don't use it. For the vast majority of users this is probably the best thing. And it's good for the ecosystem if people use shards so it doesn't hurt to advocate it.

Reply:
> In case it wasn't clear, this literally puts the line ``VERSION = {{ `shards version #{__DIR__}`.chomp.stringify }}`` into all new projects. It's not a dependency of `crystal init`, it's a dependency of every library.

Reply:
I know, this is a considerable downside. Maybe a better long-term solution would be to include a simple parser for `shards.yml` in the standard library (maybe even directly as a macro method). I've already suggested that idea in #4754 to provide metadata about the project for the docs generator. This way it wouldn't depend on the executable but read `shards.yml` directly.